components/Layout/HeadingSlackCTA/HeadingSlackCTA.module.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    top: 1rem;
+    color: inherit;
+
+    @include media-query(large) {
+        right: 0;
+    }
+
+    .logo {
+        width: 31px;
+        height: 31px;
+    }
+
+    .cta {
+        @include font-size(xs);
+        display: flex;
+        flex-direction: row;
+        margin-left: 0.5rem;
+        line-height: 1.2;
+        font-weight: $weight--extrabold;
+
+        @include media-query(large) {
+            flex-direction: column;
+        }
+
+        &__tag {
+            color: var(--color--accent);
+            font-weight: inherit;
```

Comment:
As this is placed in a strong tag, is font-weight inherit needed here?

Reply:
The strong tag has `font-weight: bold;` whereas we want to have `font-weight: extrabold;` here.